phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/library/control/LibraryFeaturePanel.java

```diff
@@ -146,6 +146,17 @@ public void createShortcut(ShortcutCreationDTO shortcutCreationDTO) {
         // get container
         // TODO: smarter way using container manager
         final String executablePath = shortcutCreationDTO.getExecutable().getAbsolutePath();
+        if (!executablePath.startsWith(getContainersPath())) {
+            Platform.runLater(() -> {
+                final ErrorDialog errorDialog = ErrorDialog.builder()
+                        .withMessage(tr("Creating shortcut to executable outside of a container is not supported"))
+                        .withOwner(getScene().getWindow())
+                        .build();
+
+                errorDialog.showAndWait();
+            });
+            return;
```

Comment:
Can we move the remaining code of this method to a new method?
I don't like the `return;` here, we could try to replace it with an `if ... else ...` syntax.
What do you think?